ethcore/engines/validator-set/reporting-config/Cargo.toml

```diff
@@ -0,0 +1,7 @@
+[package]
+description = "Validators reporting config"
+name = "validator-reporting-config"
```

Comment:
creating a new library for a single structure that is a config option for a `validator-set` library doesn't make sense. The content of this library should be a part of `validator-set` lib

Reply:
The main intention to move this into a separate module was the idea of not introducing other dependancies to the engine crate, which `validator-set` would bring with necessary to use just one struct from there. This was inspired by `call-contract` and other similar crates.

So you think it would be ok, to add `validator-set` to the Engine?

Reply:
> So you think it would be ok, to add `validator-set` to the Engine?

No, but if you use `ReportingConfig` in the construction phase of the `validator-set` engine, the `Engine` trait doesn't have to know about this `ReportingConfig` struct at all